### DISCUSSION TITLE
Sprint 0 Planning and Meeting Minutes Template

### DISCUSSION THREAD

**Discussion by vpzomtrrfrt** [cs481-ekh/f22-viz-kids#1] [Meeting Minutes]
# m/dd/2022 H:MMpm - H:MMpm [Sprint x Planning]

**Excused:** none
**Absent:** none

### Roles
**Lead:** name
**Scribe:** name
**PO:** name
**Dev:** name
**Dev:** name

### Summary
* item
* item
* item

### Sprint Goal
goal

### Action Items
- [ ] **Everyone:** work
- [ ] **Colin:** work
- [ ] **Connor:** work
- [ ] **Cory:** work
- [ ] **Javi:** work
- [ ] **Will:** work

**Comment by ConnorJacksonBSU** [comment]
# 9/11/2022 6:00pm - 7:30pm [Sprint 0 Planning]

**Excused:** none
**Absent:** none

### Summary
* Completed project setup assignment as a team
* Came up with our epics and sprint goal
* Brainstormed action items
* Created user stories and tasks

### Sprint Goal
By the end of this sprint we will be able to run a simple version of our web page to demonstrate the initial functionality. This will be a single rendered 3D object to begin with, without any other UI elements. We’ll also have basic continuous integration and a dummy test added to our testing framework.

### Action Items
- [x] **Everyone:** Install node & npm, create UXpin account
- [x] **Colin:** Init a React, Three.js, typescript hello world project
- [x] **Connor & Javi:** Create wireframe design prototype (linked in README)
- [x] **Cory:** Initial setup of GitHub workflows CI
- [x] **Will:** Initial setup of test framework and script
